Sources/SessionPublisherProtocol.swift

```diff
@@ -32,7 +32,7 @@ public protocol SessionPublisherProtocol: AnyObject {
 extension URLSession: SessionPublisherProtocol {
   public func dataTaskPublisher(for request: URLRequest) -> AnyPublisher<(data: Data, response: URLResponse), Error> {
     self.dataTaskPublisher(for: request)
-    .receive(on: DispatchQueue.main)
+    .subscribe(on: DispatchQueue.global())
```

Comment:
I don't think networking framework should decide where to subscribe and publish it's changes

Reply:
It's a design decision that I took while creating the framework. For the consumer it doesn't change anything actually but I wanted fusion to provide a hassle-free development for the consumer. Therefore all the executions happen on the background thread and the deliveries happen on the main queue.

If at any point a consumer wants to use a different thread, they are free to use `subscribe or receive`